helm_deploy/hmpps-temporary-accommodation-ui/values.yaml

```diff
@@ -68,6 +68,11 @@ generic-service:
     cloudplatform-live1-1: '35.178.209.113/32'
     cloudplatform-live1-2: '3.8.51.207/32'
     cloudplatform-live1-3: '35.177.252.54/32'
+    ark-nps-hmcts-ttp1: "195.59.75.0/24"
+    ark-nps-hmcts-ttp2: "194.33.192.0/25"
+    ark-nps-hmcts-ttp3: "194.33.193.0/25"
+    ark-nps-hmcts-ttp4: "194.33.196.0/25"
```

Comment:
We've verified that our first user is on DOM within this range so it should solve the immediate issue of smoke testing prod.

Reply:
for maintainability and clarity would it be worth incorporating "DOM1" into this somehow e.g. within the keys like:

```
ark-nps-hmcts-dom1-ttp1: "195.59.75.0/24"
...
```

Reply:
I wasn't sure if the keys had to match something downstream given that's how Create and Vary have done it. I'll see if we can set our own names in the next batch 👍